conans/test/utils/tools.py

```diff
@@ -816,6 +817,7 @@ def search(self, pattern, remote=None, assert_error=False, args=None):
         data = json.loads(self.load(".tmp.json"))
         return data
 
+
```

Comment:
why extra blank line?

Reply:
Huh, weird, no idea how it got there, good catch